dynamo/src/main/java/no/bibsys/db/TableDriver.java

```diff
@@ -78,80 +73,77 @@ public boolean tableExists(final String tableName) {
         return exists;
     }
 
-    
+
     /**
      * Return number of items in table
      * 
      * @param tableName
      * @return number of items
      */
-    
+
     public long tableSize(final String tableName) {
-    
```

Comment:
One scan does not cover the whole database. It covers max. 1MB of data. It can be used to check if the database is empty but not to estimate the size of the table.
https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Scan.html